language/move-prover/lab/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "prover-lab"
+version = "0.1.0"
+authors = ["Diem Association <opensource@diem.com>"]
+publish = false
+edition = "2018"
+license = "Apache-2.0"
+
+[dependencies]
+# diem dependencies
+move-model = { path = "../../move-model" }
+move-prover = { path = ".." }
+bytecode = { path = "../bytecode"}
+
+# FB external dependencies
+z3tracer = { git = "https://github.com/facebookincubator/smt2utils.git" }
```

Comment:
Can we use the official `z3tracer` crate? I just published `v0.8.0` on crates.io

Reply:
I tried but somehow model parsing hangs. Can this be related to missing changes in 0.8.0?

Reply:
I got 0.8.0 to work on a second try. Don't know but it appears to really take long. Will observe whether this is remains a problem, but submitting using this dep for now.